src/darker/git.py

```diff
@@ -29,19 +29,7 @@
 from darker.multiline_strings import get_multiline_string_ranges
 from darker.utils import GIT_DATEFORMAT, TextDocument
 
-if sys.version_info < (3, 8):
-
-    def shlex_join(split_command: Iterable[str]) -> str:
-        """Backport `shlex.join` for Python 3.7
-
-        :param split_command: The elements on the command line
-        :return: The command line as one string, with appropriate quoting
-
-        """
-        return " ".join(shlex.quote(arg) for arg in split_command)
-
-else:
-    shlex_join = shlex.join
+shlex_join = shlex.join
```

Comment:
Could we just use `shlex.join` directly everywhere?

Reply:
Good idea, updated!